### PR TITLE
My first branch

### PR DESCRIPTION
**Pull request by nimplay**
This is the "Hello Microverse" project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# Module1-Hello-Microverse
\ No newline at end of file
+This is my first microverse project into github.
+
+The content is a HTML and CSS file, two md files and a folder that contain linters file.
+
+Also in adition a file .gitignore that can hide a md file with the content "This file should be ignored by git" and the workflows folder with the linters.
+
+The HTML file only contain a title in h1 that say "Hello Microverse" and, the CSS file class set a font family style and color font.
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly your readme from [here](https://github.com/microverseinc/readme-template) which is provided by microverse.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+workflows
```
Comment (sja-thedude) [inline .gitignore]: - [x] Kindly replace workflows with node_modules as per the requirements of the project.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @nimplay, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on for approval but you are almost there!

first review

## Suggested changes
- [x] Kindly add all the linters from [here](https://github.com/microverseinc/linters-config/tree/master/html-css) except the readme and make sure all the linters are passing.
- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .gitignore (hunk 2 above)
Comment (nimplay) [reply .gitignore]: only I need replace the workflows folder by node_modules or I need to do something in specific?

Comment (nimplay) [reply .gitignore]: I finish all requirenment please review

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @nimplay,
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
- [x] [OPTIONAL] Pull Request should have a summary describing introduced changes in more detail, please add describe changes in the PR description 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nimplay** [comment]
Summary:
In this project I got to know in an introductory way the use of linters in my html and css projects in git and in the code editor. In addition, the necessary parameters for to write a file README in a professional way.
I also worked the creation, elimination and management of branches from the terminal of my computer.

**Review by m05e5 — APPROVED** [review]
# Hi @nimplay , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You submitted the wrong Pull Request to the wrong project title.
This project is `Hello Microverse` and you submitted it to `Portfolio: setup and mobile version skeleton` project. Please submit the right PR link to the right project title

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**